src/steamcompmgr.cpp

```diff
@@ -3333,13 +3400,41 @@ dispatch_nudge( int fd )
 	}
 }
 
+struct rgba_t
+{
+	uint8_t r,g,b,a;
+};
+
+static bool
+load_mouse_cursor( MouseCursor *cursor, const char *path )
+{
+	int w, h, channels;
+	rgba_t* data = (rgba_t*)stbi_load(path, &w, &h, &channels, STBI_rgb_alpha);
+	if (!data)
+	{
+		fprintf(stderr, "Failed to open/load cursor file\n");
+		return false;
+	}
+
+	std::transform(data, data + w * h, data, [](rgba_t x) {
```

Comment:
What's the goal of this transformation pass? Wouldn't a zero alpha result in a transparent pixel, no matter what the RGB values are?